frappe/tests/test_db.py

```diff
@@ -45,11 +46,27 @@ def test_escape(self):
 		frappe.db.escape("香港濟生堂製藥有限公司 - IT".encode("utf-8"))
 
 	def test_get_single_value(self):
-		frappe.db.set_value('System Settings', 'System Settings', 'backup_limit', 5)
-		frappe.db.commit()
+		#setup
+		fieldtypes = ['Float', 'Int', 'Percent', 'Currency', 'Data', 'Date', 'Datetime', 'Time']
+		values = [1.5, 1, 55.5, 12.5, 'Test', datetime.datetime.now().date(), datetime.datetime.now(), datetime.timedelta(hours=9, minutes=45, seconds=10)]
+		test_inputs = [{
+			'fieldtype': fieldtypes[i],
+			'value': values[i]} for i in range(len(fieldtypes))]
```

Comment:
Seems like a dict would be more suitable here? Would increase readability for sure. And you can iterate over them as

```py
		test_inputs = [{
			'fieldtype': fieldtype,
			'value': value} for fieldtype, value in values_dict.items()]
```